qiskit/visualization/state_visualization.py

```diff
@@ -233,7 +233,7 @@ def plot_bloch_vector(bloch, title="", ax=None, figsize=None, coord_type="cartes
 
 
 @deprecate_arguments({"rho": "state"})
-def plot_bloch_multivector(state, title="", figsize=None, *, rho=None, reverse_bits=False):
+def plot_bloch_multivector(state, filename=None, title="", figsize=None, *, rho=None, reverse_bits=False):
```

Comment:
New params should generally be added at the end of the params list :)

Reply:
done